paths/comments/index.yaml

```diff
@@ -44,12 +66,43 @@ x-code-samples:
 - lang: Curl
   source: |-
     curl "https://api.phrase.com/v2/projects/:project_id/keys/:key_id/comments?branch=my-feature-branch" \
-      -u USERNAME_OR_ACCESS_TOKEN
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X GET \
+      -d '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content", "locale_ids": ["someId"] }' \
+      -H 'Content-Type: application/json'
 - lang: CLI v2
   source: |-
     phrase comments list \
     --project_id <project_id> \
     --key_id <key_id> \
-    --branch my-feature-branch \
+    --data '{"branch":"my-feature-branch", "filters": ["unread", "read"], "query": "Some comment content", "locale_ids": ["someId"] }' \
```

Comment:
I believe this is a GET request, so the parameters should be passed as query params, not as JSON?

Reply:
Adjusted. This is probably something we have to check for every endpoint also, seems like it's pretty mixed atm 🤔